StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1600UnitTests.cs

```diff
@@ -689,6 +691,235 @@ public int DoSomething2()
     {
         return 0;
     }
+
+    public int DoSomething3(int a, bool b, bool? c)
+    {
+        return 0;
+    }
+
+    public int DoSomething3<TType>(int a)
+    {
+        return 0;
+    }
+
+    public IReadOnlyCollection<string> DoSomething4()
+    {
+        return null;
+    }
+
+    public List<string> DoSomething5()
+    {
+        return null;
+    }
+
+    public List<string> DoSomething6()
+    {
+        throw new System.NotImplementedException();
+        throw new System.Exception();
+    }
+}
+";
+
+            var fixedTestCode = @"
+using System.Collections.Generic;
+
+/// <summary>
+/// Test class.
+/// </summary>
+public class TestClass
```

Comment:
📝 Most values added in this test need to be updated according to other comments given previously.